### PR TITLE
support faster_rcnn for kunlun

### PR DESCRIPTION
**Pull request by cucuzg**
support faster_rcnn for kunlun
test=document_fix

### CODE REVIEW

File: configs/faster_rcnn_r50_roadsign.yml [hunk 1]
```diff
@@ -0,0 +1,238 @@
+architecture: FasterRCNN
+use_gpu: true
+max_iters: 2000
+log_iter: 1
+save_dir: output
+snapshot_iter: 500
+metric: COCO
+pretrain_weights: https://paddlemodels.bj.bcebos.com/object_detection/faster_rcnn_r50_vd_fpn_2x.tar
+weights: output/faster_rcnn_r50_vd_fpn_roadsign_coco_template/best_model
```
Comment (jerrywgz) [inline configs/faster_rcnn_r50_roadsign.yml:9]: output/faster_rcnn_r50_vd_fpn_roadsign/model_final

**Review by jerrywgz — COMMENTED** [review]
change the config file name to faster_rcnn_r50_vd_fpn_roadsign.yml

File: configs/faster_rcnn_r50_roadsign.yml [hunk 2]
```diff
@@ -0,0 +1,238 @@
+architecture: FasterRCNN
+use_gpu: true
```
Comment (QingshuChen) [inline configs/faster_rcnn_r50_roadsign.yml:2]: use_gpu： true这个不是跑在gpu上？

**Review by jerrywgz — APPROVED** [review]
LGTM

File: configs/faster_rcnn_r50_roadsign.yml (hunk 2 above)
Comment (cucuzg) [reply configs/faster_rcnn_r50_roadsign.yml]: fixed

**Pull request merged**